src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,60 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String readString = readFile(fromFileName);
+        String report = createReport(readString);
+        writeFile(toFileName, report);
+    }
+
+    private String readFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        File readFile = new File(fromFileName);
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(readFile))) {
+            String localString = bufferedReader.readLine();
+            while (!(localString == null)) {
```

Comment:
```suggestion
            while (bufferedReader.readLine() != null) {
```

Reply:
If I rewrite it, cursor read doen't move to one line below?

Reply:
I checked it. That so.
If I use 
while (bufferedReader.readLine() != null) { 
I lose 1 line and resut woul be wrong.